### DISCUSSION TITLE
eBUS adapter 3 with home assistant and vaillant eco-tec

### DISCUSSION THREAD

**Discussion by sven055** [john30/ebusd#854] [Q&A]
Hallo,
some kind of problem here. I installed the add-on in home assistant, configured my ebus 3 wifi-adapter and watched all the logs within Home Assistant, MQTT-Explorer and eBUS adapter 3 without any solution to get information’s from my vaillant eco-tec heating system. May someone can give me some hints to solve the problem…

![grafik](https://user-images.githubusercontent.com/60066796/225367844-a5631be5-6ee9-4518-b07d-a36f9daea18c.png)

![grafik](https://user-images.githubusercontent.com/60066796/225367758-cbf6b9b1-f131-453e-ad91-64ca6e3afacb.png)
![grafik](https://user-images.githubusercontent.com/60066796/225367888-69d46cf0-96d3-4e92-a080-52bf7d49f852.png)

**Comment by Koky05** [comment]
In your case I will delete `configpath` from addon configuration to use native configuration files. Right now it reads local config files and it looks like they are corrupted.
